tests/add_fee_amount.go

```diff
@@ -0,0 +1,86 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/network"
+	"github.com/ava-labs/teleporter/tests/utils"
+	localUtils "github.com/ava-labs/teleporter/tests/utils/local-network-utils"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+func AddFeeAmountGinkgo() {
```

Comment:
nit: instead of `Ginkgo` can we rename to be `Local` or `_Local`?

Reply:
`AddFeeAmount(network.Network)` can still be run against a local network by providing a `&network.LocalNetwork` as the argument. `AddFeeAmountGinkgo()` on the other hand is intended only to be used by a Ginkgo spec, so I think the current naming convention is the most accurate.